packages/shared/src/lib/featureManagement.ts

```diff
@@ -42,7 +42,15 @@ const feature = {
   ),
   extensionOverlay: new Feature('onboarding_extension_overlay', false),
   seniorContentOnboarding: new Feature('senior_content_onboarding', false),
-  plusSubscription: new Feature('plus_subscription', false),
+  plusSubscription: new Feature('plus_subscription', true),
```

Comment:
Default is wrong here :) 
Also bottom one not sure if they match current?

Reply:
woops, just had that for testing.